jsk_2016_01_baxter_apc/launch/include/segmentation_each_object_in_tote.launch

```diff
@@ -0,0 +1,67 @@
+<launch>
+  <arg name="NODELET_MANAGER" />
+  <arg name="NODELET_ID_0" />
+
+  <arg name="INPUT_CLOUD"/>
+  <arg name="INPUT_IMAGE"/>
+  <arg name="HAND" />
+
+  <group ns="$(arg NODELET_MANAGER)">
+    <rosparam subst_value="true">
+      $(arg NODELET_ID_0):
+        - name: $(arg HAND)_in_tote_clipper
+          type: jsk_pcl/AttentionClipper
+          remappings:
+            - from: ~input/points
+              to: $(arg INPUT_CLOUD)
+        - name: $(arg HAND)_in_tote_extract_pi
+          type: jsk_pcl/ExtractIndices
+          remappings:
+            - from: ~input
+              to: $(arg INPUT_CLOUD)
+            - from: ~indices
+              to: $(arg HAND)_in_tote_clipper/output/point_indices
+        - name: $(arg HAND)_in_tote_euclid_clustering
+          type: jsk_pcl/EuclideanClustering
+          remappings:
+            - from: ~input
+              to: $(arg HAND)_in_tote_extract_pi/output
+        - name: $(arg HAND)_in_tote_cpi_decomposer
+          type: jsk_pcl/ClusterPointIndicesDecomposer
+          remappings:
+            - from: ~input
+              to: $(arg HAND)_in_tote_extract_pi/output
+            - from: ~target
+              to: $(arg HAND)_in_tote_euclid_clustering/output
+    </rosparam>
+  </group>
+  <group ns="$(arg HAND)_in_tote_clipper">
+    <rosparam>
+      initial_pos: [0.75, 0, -0.25]
+      initial_rot: [0, 0, 0]
+      dimension_x: 0.25
+      dimension_y: 0.45
+      dimension_z: 0.15
+      frame_id: base
+    </rosparam>
+  </group>
+  <group ns="$(arg HAND)_in_tote_extract_pi">
+    <rosparam>
+      max_queue_size: 500
```

Comment:
i just copied from `jsk_2015_05_baxter_apc/recognition_in_hand.launch`, i'm not sure